pkg/config/signer.go

```diff
@@ -149,21 +146,17 @@ func (s *Signer) CreateSignatureHeader() (*common.SignatureHeader, error) {
 
 // createNonce generates a nonce using the crypto/rand package.
 func createNonce() ([]byte, error) {
-	nonce, err := getRandomNonce()
+	nonce := make([]byte, 24)
+
+	_, err := rand.Read(nonce)
 	if err != nil {
-		return nil, fmt.Errorf("failed to generate random nonce: %s", err)
+		return nil, fmt.Errorf("failed to get random bytes: %s", err)
```

Comment:
nit: we use `%v` just about everywhere else already in this package for errors, might as well standardize 🤷‍♂️There's really not a huge different between `%s` and `%v` so I'm fine with whichever

Reply:
done